examples/fluid/pipe_flow_2d.jl

```diff
@@ -0,0 +1,120 @@
+# TODO: Description
+using TrixiParticles
+using OrdinaryDiffEq
+
+# ==========================================================================================
+# ==== Resolution
+domain_length_factor = 0.05
+
+# Make sure that the kernel support of fluid particles at a boundary is always fully sampled
+boundary_layers = 4
+
+# Make sure that the kernel support of fluid particles at an open boundary is always
+# fully sampled.
+# Due to the dynamics at the inlets and outlets, it is recommended to use
+# `open_boundary_layers > boundary_layers`
+open_boundary_layers = 6
+
+# ==========================================================================================
+# ==== Experiment Setup
+tspan = (0.0, 2.0)
+
+# Boundary geometry and initial fluid particle positions
+domain_size = (1.0, 0.4)
+
+flow_direction = [1.0, 0.0]
+reynolds_number = 100
+const prescribed_velocity = 2.0
+
+particle_spacing = domain_length_factor * domain_size[1]
+
+boundary_size = (domain_size[1] + 2 * particle_spacing * open_boundary_layers,
+                 domain_size[2])
+
+fluid_density = 1000.0
+pressure = 1000.0
```

Comment:
Do you mean the pressure?
For this particular example, it is necessary to have a background pressure. Otherwise the suction at the outflow is to big and the simulation becomes unstable. With edac and TVF we can use higher pressure so that we don't have this "suction phenomena"

Reply:
Then please add a comment.